modules/tensor_mechanics/src/materials/ComputeThermalExpansionEigenstrainBase.C

```diff
@@ -14,6 +14,7 @@ validParams<ComputeThermalExpansionEigenstrainBase>()
   InputParameters params = validParams<ComputeEigenstrainBase>();
   params.addCoupledVar("temperature", "Coupled temperature");
   params.addCoupledVar("stress_free_temperature",
+                       0.0,
```

Comment:
Sorry for chiming in late about this. We have intentionally removed all defaults for stress free temperatures because there really isn't a single value that makes sense as a universal default, so we want to make it so that users have to specify something here. I'll submit a PR to undo this.

Reply:
I didn't mean to come off too negative there... I'm otherwise really happy with this change set.  This is one that was on our to-do list, and I'm happy to see that it's already done!

Reply:
No worries.  I can get rid of that default.  At the time i thought it was rather strange that there was no default, but if you don't want one then i'll remove.